### PR TITLE
Only use the gdas backgrounds

### PR DESCRIPTION
**Pull request by guillaumevernieres**
When running the `gfs` cycle, the marine DA prep was trying to link to the gfs background, which is not correct.

### CODE REVIEW

**Review by CoryMartin-NOAA — DISMISSED** [review]
Fine, but FYI, the current atmospheric convention is to define:
```
GDUMP=${GDUMP:-"gdas"}
```
and then use `${GDUMP}.t${gcyc}z`...

**Comment by guillaumevernieres** [comment]
> Fine, but FYI, the current atmospheric convention is to define:
> 
> ```
> GDUMP=${GDUMP:-"gdas"}
> ```
> 
> and then use `${GDUMP}.t${gcyc}z`...

OK, thanks for the feed back @CoryMartin-NOAA . I'll make the changes.

**Review by ShastriPaturi — DISMISSED** [review]

File: scripts/exgdas_global_marine_analysis_prep.py [hunk 1]
```diff
@@ -507,12 +507,12 @@ def find_clim_ens(input_date):
 # Copy initial condition
 ics_list = []
 # ocean IC's
-mom_ic_src = glob.glob(os.path.join(bkg_dir, f'{RUN}.*.ocnf003.nc'))[0]
```
Comment (CoryMartin-NOAA) [inline scripts/exgdas_global_marine_analysis_prep.py:510]: are you defining `$GDUMP` anywhere?

Comment (guillaumevernieres) [reply scripts/exgdas_global_marine_analysis_prep.py:510]: Yes @CoryMartin-NOAA , in the jjob ... Luck, or unfortunate copy/paste that became a blessing?

Comment (CoryMartin-NOAA) [reply scripts/exgdas_global_marine_analysis_prep.py:510]: ok sure, but then don't you need something like line 126?
https://github.com/NOAA-EMC/GDASApp/blob/61aa906e8e7bf68dd3db54a1e2d5f39a8b6f0224/scripts/exgdas_global_marine_analysis_prep.py#L126

Comment (guillaumevernieres) [reply scripts/exgdas_global_marine_analysis_prep.py:510]: Ha!!! I clearly didn't test these changes! Thanks @CoryMartin-NOAA .

**Review by CoryMartin-NOAA — APPROVED** [review]

**Pull request merged**